AEPCore/Sources/eventhub/Event.swift

```diff
@@ -39,7 +39,17 @@ public class Event: NSObject, Codable {
 
     /// Event description used for logging
     @objc override public var description: String {
-        return "id: \(id.uuidString) name: \(name) type: \(type) source: \(source) data: \(String(describing: data)) timestamp: \(timestamp.description) responseId: \(String(describing: responseID?.uuidString))"
+        // swiftformat:disable indent
```

Comment:
this prevents warnings from `SwiftFormat` for the indentation used in the formatting of the `Event`'s description